Core/Utilities/__init__.py

```diff
@@ -1,14 +1,3 @@
-# $HeadURL$
 """
    DIRAC.Core.Utilities package
 """
-__RCSID__ = "$Id$"
-from DIRAC.Core.Utilities.File               import makeGuid, checkGuid, getSize, getGlobbedTotalSize, getGlobbedFiles, getCommonPath, getMD5ForFiles
```

Comment:
This might give problems for extensions and should be noted in the transition guide, I think.

Reply:
clearly yes, I missed that one !